res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -10,25 +10,12 @@ var TrackEndWarning = true;
 // when SHIFT enabled and scratching with the jog wheel
 var iCutEnabled = true;
 
-// fastSeekEnabled: enable fast seek with Jog Wheel with Wheel Off and Shift ON 
-// Shift can be locked or not
-var fastSeekEnabled = true;
-
 //activate PFL of deck on track load
 var smartPFL = true;
 
 // use beatlooproll instead of beatloop
 var beatlooprollActivate = false;
```

Comment:
Say instead of rolling into a drop on the same track, you bring in another track instead, and you want to leave the other track where it is instead of having it slip past when you stop the loop.

I suppose there are other ways around that though..